src/vcModals.cpp

```diff
@@ -548,9 +548,9 @@ void vcModals_DrawWelcome(vcState *pProgramState)
       {
         if (localOrServerProject == 0) // local
         {
-          // TODO: CONFIRM FILE OVERRIDE IF EXISTS
-
-          result = vcProject_CreateFileScene(pProgramState, pProjectPath, pProgramState->modelPath, zoneCustomSRID);
+          udFilename exportFilename(pProjectPath);
+          vcProject_AutoCompletedName(&exportFilename, pProjectPath, pProgramState->modelPath);
+          result = vcProject_CreateFileScene(pProgramState, exportFilename, pProgramState->modelPath, zoneCustomSRID);
```

Comment:
I think the intent here was to call `vcModals_OverwriteExistingFile` instead of failing, so the `errorFileExists` error message isn't required now.

Reply:
I saw in vcProject_CreateFileScene that it called the function vdkProject_CreateInFile, which will create an empty, only local vdkProject instance and return false if file exists. I could change the current behaviour of vcProject_CreateFileScene if you sure. And nowhere in vcProject_CreateFileScene will return vE_NotSupported now, however I can add it back too.

Reply:
Overwriting existing files is currently not supported,